src/bptl/work_units/kadaster/migrations/0001_create_brt_service.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.14 on 2020-12-07 10:50
+
+from django.db import migrations
+
+from zgw_consumers.constants import APITypes, AuthTypes
+
+from bptl.tasks.registry import register
+
+from ..tasks import retrieve_openbare_ruimten
+from ..utils import ALIAS
+
+API_ROOT = "https://brt.basisregistraties.overheid.nl/api/v2"
```

Comment:
`zgw_consumers.Service` always adds a trailing slash to `api_root`
```suggestion
API_ROOT = "https://brt.basisregistraties.overheid.nl/api/v2/"
```